modules/core/src/effects/lighting/lighting-effect.ts

```diff
@@ -144,16 +147,23 @@ export default class LightingEffect implements Effect {
 
     // when not rendering to screen, turn off lighting by adding empty light source object
     // lights shader module relies on the `lightSources` to turn on/off lighting
-    parameters.lightSources = {
+    const lightingProps = {
```

Comment:
Recommend adding an `} as const satisifies LightingProps` at the end so we catch mistakes here.